src/wasm-lib/kcl/tests/assembly_non_default_units/input.kcl

```diff
@@ -0,0 +1,10 @@
+@settings(defaultLengthUnit = in)
+
+
+// Use a default unit that isn't the default of mm.
```

Comment:
Is it correct that all the default units are in? I'd expect one to be mm, but perhaps mm comes from the default-default?

Reply:
Right. `mm` comes from the default default. In the robot kcl-example, every file was set to `in`.